lib/router/transition.js

```diff
@@ -16,7 +16,7 @@ import { trigger, slice, log, promiseLabel } from './utils';
   @param {Object} error
   @private
  */
-function Transition(router, intent, state, error) {
+function Transition(router, intent, state, error, previousTransition) {
```

Comment:
on more detailed review, it looks like `transition` is the only property that would be added to the closure.  It is a little weird to do this in a condition.

I'd likely make a function that took `transition`as an arg that returned a error handler.  This limits the scope and shape of the closure context.